tests/reconfig/results/1.0.0/1.0.0.md

```diff
@@ -0,0 +1,78 @@
+# Reconfiguration testing Results
+
+<!-- TOC -->
+- [Reconfiguration testing Results](#reconfiguration-testing-results)
+  - [Test environment](#test-environment)
+  - [Results Tables](#results-tables)
+    - [NGINX Reloads and Time to Ready](#nginx-reloads-and-time-to-ready)
+    - [Event Batch Processing](#event-batch-processing)
+  - [NumResources -> Total Resources](#numresources---total-resources)
+  - [Observations](#observations)
+<!-- TOC -->
+
+## Test environment
+
+GKE cluster:
+
+- Node count: 3
+- Instance Type: e2-medium
+- k8s version: 1.27.3-gke.100
+- Zone: us-central1-c
+- Total vCPUs: 6
+- Total RAM: 12GB
+- Max pods per node: 110
+
+NGF deployment:
+
+- NGF version: edge - git commit 29b45e38bacd7c4f22834938105e3cda4f29f6d1
+- NGINX Version: 1.25.2
+
+## Results Tables
+
+### NGINX Reloads and Time to Ready
+
+| Test number | NumResources | TimeToReadyTotal (s) | TimeToReadyAvgSingle (s) | NGINX reloads | NGINX reload avg time (ms)|
```

Comment:
Is it possible to list the reload time distribution here (like you did with event batch processing) without rerunning the test?

If not, that's ok 🙂

Reply:
Yup! I saved the logs 🙂

Reply:
Just re-running one of the tests.